airflow/models/__init__.py

```diff
@@ -1400,6 +1403,14 @@ def signal_handler(signum, frame):
 
                 task_copy.post_execute(context=context, result=result)
 
+                end_time = time.time()
+                duration = end_time - start_time
+                Stats.timing(
+                    '{dag_id}.{task_id}.duration'.format(
```

Comment:
Can you update the documentation in file `docs/metrics.rst`?   I would be happy if you would also add other missing keys. 
Reference: 
My WIP PR that add missing keys - 
https://github.com/PolideaInternal/airflow/pull/79/commits/bed9e3bd26a242abb5a0ef1bf35e7d44ab1426f1

Reply:
I have updated the docs!

Reply:
@mik-laj you want me to take over your PR?

Reply:
if you have time then I would be happy. However, I do not require it. I gave the link to show that other things could be corrected in the documentation. I do not care about finishing my PR.